api-tests/specs/createTrialSite/createNewTrialSite.js

```diff
@@ -155,7 +155,7 @@ describe('As a user with Create Trial Sites permission I want to have the system
             method: 'POST',
             body: JSON.stringify(requests.missingParent),
         })
-        expect(fetchResponse11.status).to.equal(HttpStatus.FORBIDDEN)
+        expect(fetchResponse11.status).to.equal(HttpStatus.UNPROCESSABLE_ENTITY)
```

Comment:
while testing noticed that a 403 was being thrown when a parentSiteId was passed - this should be a 422 - I changed the code so that the error occurs when validating and not when doing auth.